### DISCUSSION TITLE
HTTPS enabled exporters

### DISCUSSION THREAD

**Discussion by pedro-leitao** [open-telemetry/opentelemetry-java#3908] [Q&A]
Hello.

Is there a way to export to a https enabled Jaeger endpoint ?

Also, is anyone working on a UDP Jaeger exporter ?

Regards.

**Comment by jkwatson** [comment]
I believe our jaeger-thrift exporter will handle https, yes. I don't know of anyone working on a UDP Jaeger exporter, no.
